dap_aria_mapping/getters/patents.py

```diff
@@ -30,6 +30,28 @@ def get_patent_entities() -> Mapping[str, Mapping[str, Union[str, str]]]:
         download_as="dict",
     )
 
+def get_patent_topics(tax: str = 'cooccur', level: int = 1) -> Dict:
+    """gets patent ids tagged with topics from the specified taxonomy
+    at a given level"
+
+    Args:
+        tax (str, optional): taxonomy labels to use. Defaults to 'cooccur'.
+            PIPELINE HAS CURRENTLY ONLY BEEN RUN FOR COOCURRENCE SO THIS IS THE ONLY OPTION
```

Comment:
Let's not risk forgetting we have this in the docstrings, ie. not push it to dev?

Reply:
Why not have this in dev? Once we decide on a taxonomy and re-run the pipeline on the taxonomy we're moving forward with we can remove it, but I think it's helpful for now?